src/NServiceBus.Testing/Saga.cs

```diff
@@ -32,6 +34,11 @@ internal Saga(T saga)
             saga.Entity.Originator = "client";
         }
 
+        Saga()
+        {
+            timeoutMethodInfo = typeof(Saga<T>).GetMethod("HandleTimeout", BindingFlags.Instance | BindingFlags.NonPublic);
```

Comment:
Only need to find the method once. If I put it in `WhenSagaTimesOut` it would be called every time which would be more expensive than it needs to be.

Happy to move though